libs/application/templates/public-debt-payment-plan/src/lib/messages/externalData.ts

```diff
@@ -53,5 +53,25 @@ export const externalData = {
         'Símanúmer, netfang. Upplýsingar um símanúmer eða netfang er hægt að uppfæra á vefsíðu island.is ef þess þarf.',
       description: 'User Profile Subtitle',
     },
+    paymentEmployerTitle: {
+      id: `pdpp.application:section.externalData.paymentEmployerTitle`,
+      defaultMessage: 'Upplýsingar frá skattinum',
+      description: 'External data section payment plan title',
```

Comment:
Probably want to translate this and the strings below before you run extract strings otherwise youll have to do it in contentful and thats more annoying.